packages/web3-eth-personal/types/index.d.ts

```diff
@@ -0,0 +1,89 @@
+/*
+    This file is part of web3.js.
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file index.d.ts
+ * @author Huan Zhang <huanzhang30@gmail.com>,
+ * @author Josh Stevens <joshstevens19@hotmail.co.uk>
+ * @date 2018
+ */
+
+import * as net from 'net';
+import {
+    AbstractWeb3Module,
+    AccountsBase,
+    provider,
+    RLPEncodedTransaction,
+    TransactionConfig,
+    Web3ModuleOptions
+} from 'web3-core';
+
+export class Personal extends AbstractWeb3Module {
+    constructor(
+        provider: provider,
+        net?: net.Socket | null,
+        options?: Web3ModuleOptions,
+        accounts?: AccountsBase | null
+    );
+
+    newAccount(
+        password: string,
+        callback?: (error: Error, address: string) => void
+    ): Promise<string>;
+
+    sign(
+        dataToSign: string,
+        address: string,
+        password: string,
+        callback?: (error: Error, signature: string) => void
+    ): Promise<string>;
+
+    ecRecover(
+        dataThatWasSigned: string,
+        signature: string,
+        callback?: (error: Error, address: string) => void
+    ): Promise<string>;
+
+    signTransaction(
+        transactionConfig: TransactionConfig,
+        password: string,
+        callback?: (
+            error: Error,
+            RLPEncodedTransaction: RLPEncodedTransaction
+        ) => void
+    ): Promise<RLPEncodedTransaction>;
+
+    sendTransaction(
+        transactionConfig: TransactionConfig,
+        password: string,
+        callback?: (error: Error, transactionHash: string) => void
+    ): Promise<string>;
+
+    unlockAccount(
+        address: string,
+        password: string,
+        unlockDuration: number,
+        callback?: (error: Error) => void
+    ): Promise<boolean>;
+
+    lockAccount(
+        address: string,
+        callback?: (error: Error, success: boolean) => void
+    ): Promise<boolean>;
+
+    getAccounts(
```

Comment:
getAccounts is not documented in the docs of the 1.x branch (could you do this/copy it for me or could you create an issue for it?; same to lockAccount, sendTransaction, and importRawKey.)

Reply:
sure have done that for you.